documentation/asciidoc/computers/raspberry-pi-5/introduction.adoc

```diff
@@ -8,7 +8,7 @@ image::images/pi5-labelled.jpg[alt="Labelled diagram of the Raspberry Pi 5",widt
 
 Featuring a 64-bit quad-core Arm Cortex-A76 processor running at 2.4GHz, Raspberry Pi 5 delivers a ×2 to ×3 increase in CPU performance relative to Raspberry Pi 4. Alongside a substantial uplift in graphics performance from an 800MHz VideoCore VII GPU; dual 4Kp60 display output over HDMI; and state-of-the-art camera support from a rearchitected Raspberry Pi Image Signal Processor, it provides a smooth desktop experience for consumers, and opens the door to new applications for industrial customers.
 
-For the first time, this is a full-size Raspberry Pi computer using silicon built in-house at Raspberry Pi. The xref:../microcontrollers/rp1.adoc[RP1] provides the bulk of the I/O capabilities for Raspberry Pi 5, and delivers a step change in peripheral performance and functionality. Aggregate USB bandwidth is more than doubled, yielding faster transfer speeds to external UAS drives and other high-speed peripherals; the dedicated two-lane 1Gbps MIPI camera and display interfaces present on earlier models have been replaced by a pair of four-lane 1.5Gbps MIPI transceivers, tripling total bandwidth, and supporting any combination of up to two cameras or displays; peak SD card performance is doubled, through support for the SDR104 high-speed mode; and for the first time the platform exposes a single-lane PCI Express 2.0 interface, providing support for high-bandwidth peripherals.
+For the first time, this is a full-size Raspberry Pi computer using silicon built in-house at Raspberry Pi. The xref:../microcontrollers/rp1.adoc[RP1] provides the bulk of the I/O capabilities for Raspberry Pi 5, and delivers a steep change in peripheral performance and functionality. Aggregate USB bandwidth is more than doubled, yielding faster transfer speeds to external UAS drives and other high-speed peripherals; the dedicated two-lane 1Gbps MIPI camera and display interfaces present on earlier models have been replaced by a pair of four-lane 1.5Gbps MIPI transceivers, tripling total bandwidth, and supporting any combination of up to two cameras or displays; peak SD card performance is doubled, through support for the SDR104 high-speed mode; and for the first time the platform exposes a single-lane PCI Express 2.0 interface, providing support for high-bandwidth peripherals.
```

Comment:
I'm not sure what's wrong with "step change" it's a fairly common phrase?

Reply:
Apparently, you're right, but it's a chiefly UK English phrase. I had tried to check for that beforehand by looking up the word "step" in Cambridge Dictionary, but I should've looked up the whole phrase instead of just the word. In US English, we say "steep change," which has the same meaning but adds an "E" and changes the pronunciation. I had never seen the phrase "step change," so to me it looked like a typo. It also sounded like something that would mean a smaller change, like you're just taking itty bitty little steps.

As Raspberry Pi is a UK company and this is a primarily UK English documentation according to the style guide, I suppose it's fair to leave it as "step." Today I learned! Thank you.